common-api/src/test/java/org/pmiops/workbench/db/model/CommonStorageEnumsTest.java

```diff
@@ -4,103 +4,86 @@
 import static com.google.common.truth.Truth.assertWithMessage;
```

Comment:
Is it worth even fiddling with `common-api` if it's about to get reabsorbed?

Reply:
Perhaps it wasn't when I messed with it last week.  But now that I've done so, I have broken it and hence this fix.  Oh well.

Reply:
Can we just move (F6 in IJ) all the public methods from CommonStorageEnums to DbStorageEnums and delete CommonStorageEnums?

Reply:
no.  they are in common-api.